File: CS/WindowsApplication3/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.Utils.Win;
using DevExpress.XtraEditors.Popup;
using DevExpress.XtraEditors;

namespace WindowsApplication3
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void checkedComboBoxEdit1_Properties_Popup(object sender, EventArgs e)
        {
            PopupContainerForm popupForm = ((IPopupControl)sender).PopupWindow as PopupContainerForm;
            popupForm.KeyPreview = true;
            popupForm.KeyUp += new KeyEventHandler(popupForm_KeyUp);
        }

        void popupForm_KeyUp(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Tab) {
                (sender as PopupContainerForm).ClosePopup();
            }
            Form f = (sender as PopupContainerForm).OwnerEdit.FindForm();
            Control c = f.GetNextControl((sender as PopupContainerForm).OwnerEdit, !e.Shift);
            while (!c.CanFocus) 
            {
                c = f.GetNextControl(c, !e.Shift);
            }
            c.Focus();
        }
    }
}

File: VB/WindowsApplication3/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Utils.Win
Imports DevExpress.XtraEditors.Popup
Imports DevExpress.XtraEditors

Namespace WindowsApplication3
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub checkedComboBoxEdit1_Properties_Popup(ByVal sender As Object, ByVal e As EventArgs) Handles checkedComboBoxEdit1.Properties.Popup
			Dim popupForm As PopupContainerForm = TryCast((CType(sender, IPopupControl)).PopupWindow, PopupContainerForm)
			popupForm.KeyPreview = True
			AddHandler popupForm.KeyUp, AddressOf popupForm_KeyUp
		End Sub

		Private Sub popupForm_KeyUp(ByVal sender As Object, ByVal e As KeyEventArgs)
			If e.KeyCode = Keys.Tab Then
                CType(sender, PopupContainerForm).ClosePopup()
			End If
			Dim f As Form = (TryCast(sender, PopupContainerForm)).OwnerEdit.FindForm()
			Dim c As Control = f.GetNextControl((TryCast(sender, PopupContainerForm)).OwnerEdit, (Not e.Shift))
			Do While Not c.CanFocus
				c = f.GetNextControl(c, (Not e.Shift))
			Loop
			c.Focus()
		End Sub
	End Class
End Namespace